integration-tests/contract-security-wrong-field-annotations.spec.ts

```diff
@@ -0,0 +1,275 @@
+ * called, if annotation is done in a wrong way).
+
+ * With the current behavior of annotations, we sometimes suggest in our smart contract assessment to not use field annotations, 
+ * when calling other contracts. E.g. if a DEX crafts his calls to external contract and does not use any annotations 
+ * (https://github.com/InferenceAG/TezosSecurityBaselineChecking/blob/master/testcases/TC-009/execute_test.sh#L36), 
+ * the DEX is still able to interact e.g. with smart contracts which are not FA1.2/FA2 compatible, because they are using wrong annotations 
+ * (e.g. "to" instead of "to_" for FA2 transfer entrypoints).
+ *  
+ * An example of the test contract is here: https://better-call.dev/ithacanet/KT1C4NcV4ebR6gS7SzbavkkSx3HYhWTBPVJ9/operations
+ * 
+ * The contract used has six entrypoints:
+ * 
+ * entrypoint:          parameter:
+ * 
+ * add                  (pair %add (nat %valueA) (nat %valueB))
+ * addNoAnnot           (pair %addNoAnnot nat nat)
+ * addWrongAnnot        (pair %addWrongAnnot (nat %a) (nat %b))
+ * 
+ * call                 (pair %call (string %entrypoint) (pair %param (nat %valueA) (nat %valueB)))
+ * callNoAnnot          (pair %callNoAnnot (string %entrypoint) (pair %param nat nat))
+ * callWrongAnnot       (pair %callWrongAnnot (string %entrypoint) (pair %param (nat %a) (nat %b)))
+ * 
+ * The test checks 9 cases: For each call entrypoint the tests call each of the three add entrypoints.
+ * Its not entriely clear what should happen in each case. 
+ * 
+ * For example, 
+ *    the first case calls "add" and passes nat %valueA and nat %valueB to CONTRACT %add (pair (nat %valueA) (nat %valueB)), so the annots and params match 
+ *    the second case calls "addNoAnnot" and passes nat and nat to CONTRACT %addNoAnnot (pair (nat %valueA) (nat %valueB)), and here the  annots and params 
+ *    do not match because there are no annots.
+ *    the third case calls "addWrongAnnot" and passes nat a and nat b (where a and b are the annots) to CONTRACT %add (pair nat nat) and here the annots 
+ *    and params do not match.
+ * 
+ *    When run the tests get unexpected exceptions such as 
+ *        [a] Value is not a number: undefined
+ *        [valueA] Value is not a number: undefined
+ * 
+ *  With Jakarta there is a change regarding Annotations: https://tezos.gitlab.io/protocols/013_jakarta.html#michelson. 
+ *  This means the sub testcases 3 & 7 in TC-009 will behave differently in Jakarta.
+ *        
+ */
+
+CONFIGS().forEach(({ lib, rpc, setup, protocol }) => {
+  const Tezos = lib;
+  const mondaynet = protocol === Protocols.ProtoALpha ? test: test.skip;
+
+  describe(`Test contracts to verify wrong field annotations are leading to failed transactions using: ${rpc}`, () => {
+    beforeEach(async (done) => {
+      await setup();
+      done();
+    });
+
+    mondaynet('Verify annotation combinations on ithacanet', async () => {
```

Comment:
`Verify annotation combinations on ithacanet` should be mondaynet

Reply:
fixed.